pool-interface/src/reserve_config.rs

```diff
@@ -22,6 +22,8 @@ pub struct ReserveConfigurationMap {
     pub reserved: BytesN<1>,
     //bit 64-79: reserve factor
     pub reserve_factor: u32,
+    // A value between 0 and 100 % specifies what fraction of the underlying asset counts toward the portfolio collateral value.
```

Comment:
Do we need to specify the bit gap to be consistent with the declarations above?

Reply:
No we don't, because it is old comments with 'bit'. We'll delete it soon when actualize ReserveConfigurationMap.